xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -1429,21 +1442,75 @@ public void handleRdsResponse(
         Collections.<String>emptySet(), versionInfo, nonce, errors);
   }
 
-  private static RdsUpdate processRouteConfiguration(
+  private static List<VirtualHost> processRouteConfiguration(
```

Comment:
I'd keep the return type of `processRouteConfiguration()` to keep consistency with other resources handlers / `process*` functuions:

- `handleLdsResponse()` -> `LdsUpdate processClientSideListener()` / `LdsUpdate processServerSideListener`
- `handleCdsResponse()` -> `CdsUpdate parseCluster()` (hm, noticed  this name is out of pattern. will fix it)
- `handleRdsResponse()` -> `RdsUpdate processRouteConfiguration()`
- `handleEdsResponse()` -> `EdsUpdate processClusterLoadAssignment()`

I think effort worth for the sake of the predictability and fulfilling code reader's expectations. Even if `processRouteConfiguration` simply returns `new RdsUpdate(parseVirtualHosts(...))`

Reply:
`CdsUpdate parseCluster()` got renamed in https://github.com/grpc/grpc-java/commit/fa4b980e07400a66cfaf3693785f873031b20482#diff-6b019eb1d3518381d336168847890513a235ea4974fcb3c76a2e9ace43f1d460R828.

Reply:
Created PR to rename parseCluster() back to processCluster(): #8802.

Reply:
These methods have totally different sets/types of arguments though, only their return types and their callers have a pattern. I change the name and return type for the sake of reader's "predictability".

Reply:
Thank you!